huaweicloud/resource_huaweicloud_vbs_backup_v2.go

```diff
@@ -127,11 +127,13 @@ func resourceVBSBackupTagsV2(d *schema.ResourceData) []backups.Tag {
 func resourceVBSBackupV2Create(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*config.Config)
 	vbsClient, err := config.VbsV2Client(GetRegion(d, config))
-
 	if err != nil {
 		return fmtp.Errorf("Error creating huaweicloud vbs client: %s", err)
 	}
 
+	// for job APIs: update the endpoint of vbsClient
```

Comment:
as the description, seems this update should be after the Create API call.

Reply:
**backups.GetJobEntity**  will copy a new client from vbsClient, and update the ResourceBase item, so we can update it before calling backups.GetJobEntity.

But I will update the Endpoint after the Create operation to easy understand.